share/vss/vss.go

```diff
@@ -34,19 +34,29 @@ import (
 	"encoding/binary"
 	"errors"
 	"fmt"
+	"hash"
 	"reflect"
 
 	"github.com/dedis/crypto"
-	"github.com/dedis/crypto/util/random"
 	"github.com/dedis/crypto/share"
 	sign "github.com/dedis/crypto/sign/schnorr"
+	"github.com/dedis/crypto/util/random"
 	"github.com/dedis/protobuf"
 )
 
+// NOTE: This suite is almost the same as the one defined in abstract/group
+// => again another place where code is repeated and adds a new abstraction to
+// take care of.
+type Suite interface {
```

Comment:
Bryan's comment to this was that if you only depend on the `schnorr.Suite`, you can do

```
type Suite schnorr.Suite
```

and be done.